tests/framework/installer/installer.go

```diff
@@ -45,7 +45,7 @@ const (
 var (
 	// ** Variables that might need to be changed depending on the dev environment. The init function below will modify some of them automatically. **
 	baseTestDir       string
-	forceUseDevices   = false
+	forceUseDevices   = true
```

Comment:
Was this just for local debugging? I thought we wouldn't want to force using devices in all cases.

Reply:
This is what finally got the integration tests to run with devices. I think this has to be set to `true`.

Reply:
How about if we always set `useAllDevice` to true and get rid of this flag to force? We can force it without the flag. 

Since we still have this in the cluster CR, we will always get a filestore OSD as well, which is how the tests would be succeeding if no devices are found. Could we remove this if devices are detected? For example, we could remove this directory OSD if `IsAdditionalDeviceAvailableOnCluster()` returns true? But maybe we do this as a separate PR.
```
    directories:
    - path: ` + settings.DataDirHostPath + /* simulate legacy fallback osd behavior so existing tests still work */ `
```